source/retryable-reads/tests/listCollectionNames-serverErrors.yml

```diff
@@ -0,0 +1,136 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "coll"
+
+data: []
+minServerVersion: '4.1.6' # 4.2
+
+tests:
+    -
+        description: "ListCollectionNames succeeds after InterruptedAtShutdown"
+        clientOptions:
+            retryReads: true
+        failPoint: &failCommand_failPoint
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data: { failCommands: [listCollections], errorCode: 11600 }
+        operation:  &retryable_operation
+            name: "listCollectionNames"
+            object: database
+            result:
+                error: false
```

Comment:
If we're not asserting anything, then simply omitting both `error` (assuming you end up moving it to `operation.error`) _and_ `result` should be fine. This goes for other tests where you may not want to assert anything other than the error condition as well (e.g. GridFS, change streams).